test/tests/functional/pbs_reservations.py

```diff
@@ -616,20 +616,27 @@ def test_future_resv_conflicts_running_job(self):
         resv from getting confirmed.
         """
 
+        vnode_val = None
+        if self.mom.is_cpuset_mom():
+            vnode_val = '1:ncpus=1:vnode=' + self.server.status(NODE)[1]['id']
+
         now = int(time.time())
         # Submit a job but do not specify walltime, scheduler will consider
         # the walltime of such a job to be 5 years
         a = {'Resource_List.select': '1:ncpus=1',
              'Resource_List.place': 'excl'}
+        if self.mom.is_cpuset_mom():
+            a['Resource_List.select'] = vnode_val
+
         j = Job(TEST_USER, attrs=a)
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
         # Submit a reservation that will start after the job starts running
-        a = {'Resource_List.select': '1:ncpus=1',
-             'Resource_List.place': 'excl',
-             'reserve_start': now + 360,
+        b = {'reserve_start': now + 360,
              'reserve_end': now + 3600}
+        a.update(b)
```

Comment:
we can directly add reserve_start/end to a like we add Resource_List.select on line 629.